zingolib/src/lightclient.rs

```diff
@@ -2295,12 +2295,12 @@ mod tests {
         Runtime::new().unwrap().block_on(async move {
             let addresses = lc.do_addresses().await;
             assert_eq!(
-                "zs1q6xk3q783t5k92kjqt2rkuuww8pdw2euzy5rk6jytw97enx8fhpazdv3th4xe7vsk6e9sfpawfg"
+                "zregtestsapling1etnl5s47cqves0g5hk2dx5824rme4xv4aeauwzp4d6ys3qxykt5sw5rnaqh9syxry8vgxr7x3x4"
```

Comment:
If this address always maps from `CHIMNEY_BETTER_SEED` (in regtest mode), then we should put it in some reasonable location adjacent to the seed-phrase in the test vector crate so we can reuse and share it.